okhttp-urlconnection/src/test/java/okhttp3/OkUrlFactoryTest.java

```diff
@@ -148,6 +149,30 @@ public void setInstanceFollowRedirectsFalse() throws Exception {
     assertResponseCode(connection, 302);
   }
 
+  @Test
+  public void testURLFilter() throws Exception {
```

Comment:
Wanna test redirect from HTTPS to HTTP ? I think we have some tests like that lying around somewhere!